vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -257,6 +257,24 @@ func (r RefCountsMap) Decr(vol string) (uint, error) {
 	}
 	return rc.count, nil
 }
+// check if volume with source as mount_source belongs to vsphere plugin
+func matchNameforVMDK(mount_source string) bool {
+	var managedPluginMountStart string = "/var/lib/docker/plugins/"
```

Comment:
nit: I suspect that just  `managedPluginMountStart  := "/var/lib/docker/plugins/"` will work equally fine